safe_web_api_playground/static/js/code-snippets/safeMutableData.js

```diff
@@ -21,27 +21,27 @@ module.exports = {
     newPrivate: async () => {
       // Generate nonce with generated with safeCrypto.generateNonce
 
-      // name and secKey must be 32 bytes.
+      // hashedString and secKey must be 32 bytes.
       // You can use safeCrypto.sha3Hash to generate a 32 byte hash based on the string you input
 
-      let name = 'name-private-0101010101010101010';
+      // Why a secret key? It's used to encrypt data.
+
       let secKey = 'secret-key-010101010101010101010';
 
       try {
-        mdHandle = await window.safeMutableData.newPrivate(appHandle, name, 15001, secKey, nonce)
+        mdHandle = await window.safeMutableData.newPrivate(appHandle, hashedString, 15001, secKey, nonce)
```

Comment:
I'm not sure we should use `hashedString` as the name for the XoR name/address of a MD, using a hash function to generate a XoR name is an option but it's not mandatory, you can set any address you want as long as it's 32 bytes long.

Reply:
This is a convenience. You'll see in the `safeCrypto.sha3hash` that when the function is run it's saved to the `hashedString` variable.
Otherwise, the user will have to count out 32 characters.